docs/vocascan-server/customPages.md

```diff
@@ -101,6 +105,25 @@ pages: {
 ...
 ```
 
+### **Environment variables**
+
+An ENV config example could look like this.
+
+```env
```

Comment:
I dont think that this example is necessary :D - but if you already have it, you can may add a details tag around it, so it doesnt fill the page that much.

Reply:
This is already in a tabs section. You can decide whether you want to see the config file or the env options

Reply:
ok. then its ok.